src/plugins/intel_cpu/src/transformation_pipeline.cpp

```diff
@@ -222,11 +222,6 @@ void Transformations::PreLpt(const std::vector<ov::element::Type>& defaultPrecis
     manager.register_pass<ov::pass::ConvertNMS3ToNMS9>();
     manager.register_pass<ov::pass::ConvertNMS4ToNMS9>();
     manager.register_pass<ov::pass::ConvertNMS5ToNMS9>();
-    manager.register_pass<ov::pass::ConvertNMS9ToNMSIEInternal>();
-    manager.register_pass<ov::pass::Validate>();
-    manager.register_pass<ov::pass::ConvertMulticlassNmsToMulticlassNmsIE>();
-    manager.register_pass<ov::pass::Validate>();
-    manager.register_pass<ov::pass::ConvertMatrixNmsToMatrixNmsIE>();
     manager.register_pass<ov::pass::Validate>();
```

Comment:
I think we can remove this pass as well because it was added for `ov::pass::ConvertMatrixNmsToMatrixNmsIE`. Could you verify this moment please?
Thank you!

Reply:
thanks. removed.